opendatahub-io/odh-manifests
# Add monitoring to Trino

This PR implements additional k8s objects to add prometheus monitoring capabilities in Trino. This change will require the new `quay.io/opendatahub/trino:361` image as it contains the prometheus library to export metrics.

## Testing instructions
* Deploy the trino/base manifest in a cluster
  * You can use the command `kustomize build trino/base | {oc|kubectl} apply -n <namespace> -f -`
* Check if no pods are in `CrashLookbakOff` or `Error` state
* Check if the trino-service Service exposes the ports 8080 and 8082 (the last one is the prometheus port)
* As an optional step, create a route for trino-service Service and access the hostname. You should see the prometheus metrics.
